qiskit/providers/ibmq/managed/ibmqjobmanager.py

```diff
@@ -0,0 +1,196 @@
+
+        job_set = ManagedJobSet(name=name)
+        job_set.run(experiment_list, backend=backend, executor=self._executor, shots=shots)
+        self._job_sets.append(job_set)
+
+        return job_set
+
+    def _split_experiments(
+            self,
+            experiments: Union[List[QuantumCircuit], List[Schedule]],
+            backend: IBMQBackend,
+            max_experiments_per_job: Optional[int] = None
+    ) -> List[Union[List[QuantumCircuit], List[Schedule]]]:
+        """Split a list of experiments into sublists.
+
+        Args:
+            experiments: Experiments to be split.
+            backend: Backend to execute the experiments on.
+            max_experiments_per_job: Maximum number of experiments to run in each job.
+
+        Returns:
+            A list of sublists of experiments.
+        """
+        if hasattr(backend.configuration(), 'max_experiments'):
+            backend_max = backend.configuration().max_experiments
+            chunk_size = backend_max if max_experiments_per_job is None \
+                else min(backend_max, max_experiments_per_job)
+        elif max_experiments_per_job:
+            chunk_size = max_experiments_per_job
+        else:
+            return [experiments]
+
+        return [experiments[x:x + chunk_size] for x in range(0, len(experiments), chunk_size)]
+
+    def report(self, detailed: bool = True) -> str:
+        """Return a report on the statuses of all jobs managed by this manager.
+
+        Args:
+            detailed: True if a detailed report is be returned. False
+                if a summary report is to be returned.
+
+        Returns:
+            A report on job statuses.
+        """
+        job_set_statues = [job_set.statuses() for job_set in self._job_sets]
+        flat_status_list = [stat for stat_list in job_set_statues for stat in stat_list]
+
+        report = ["Summary report:"]
+        report.extend(format_status_counts(flat_status_list))
+
+        if detailed:
+            report.append("\nDetail report:")
+            for i, job_set in enumerate(self._job_sets):
+                report.append(("  Job set {}:".format(job_set.name())))
+                report.extend(format_job_details(
+                    job_set_statues[i], job_set.managed_jobs()))
+
+        return '\n'.join(report)
+
+    def job_set(self, name: str) -> Optional[ManagedJobSet]:
+        """Returns the managed job set with the specified name.
+
+        If multiple sets with the same name is found, the first set is returned.
+
+        Args:
+             name: Name of the managed job set.
+
+        Returns:
+            A set of managed jobs or ``None`` if no named set found.
+        """
+        for job_set in self._job_sets:
+            if job_set.name() == name:
+                return job_set
+        return None
+
+    @classmethod
+    def previous_job_set(
+            cls,
+            name: str,
+            provider: AccountProvider,
```

Comment:
Unrelated to this PR, but I'm still a bit confused by Account vs. Provider. What is `AccountProvider`? Is this what previously was `SingleProvider`? I thought Account was supposed to be all of the providers.

Reply:
An `AccountProvider` controls access by hub/group/project. An account can have multiple providers (e.g. if you belong in multiple projects).

Reply:
So one "AccountProvider" is a provider from one "h/g/p"?

Reply:
Yes!